### PR TITLE
Heap

### PR DESCRIPTION
**Pull request by bgarnaat**

### CODE REVIEW

File: src/Heap.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#  _*_coding:utf-8 _*_
+"""Create a heap data structure."""
+from __future__ import division
+import math
```
Comment (joelstanner) [inline src/Heap.py:4]: Avoid importing modules if they are not being used.

File: src/Heap.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+class Heap(object):
+    """Create a heap data structure."""
+
+    def __init__(self):
+        """Init method for Heap class."""
+        self.hl = []
```
Comment (joelstanner) [inline src/Heap.py:12]: From the assignment description: "The constructor for your heap should default to creating an empty heap, but allow for creating a populated heap given an iterable as an input." This constructor does not accept any arguments. Also, `hl` is an ambiguous variable name. Avoid that when possible (`heap_list` would be better).

File: src/test_Heap.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    from Heap import Heap
+    hl = Heap()
+    hl.push(data[0])
+    hl.push(data[1])
+    hl.push(data[2])
+    assert hl.hl[hl.get_left(0)] == data[1]
+
+
+def test_get_right():
+    """Test get_left method."""
```
Comment (joelstanner) [inline src/test_Heap.py:44]: Having docstrings that don't mean anything is not useful, especially when they're incorrect such as this one and the one below.

File: src/test_Heap.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+def test_pop():
+    """Test pop method."""
+    from Heap import Heap
+    hl = Heap()
+    hl.push(data[2])
+    hl.push(data[1])
+    hl.push(data[3])
+    assert hl.pop() == data[2]
+
```
Comment (joelstanner) [inline src/test_Heap.py:72]: Make sure to add edge case tests, such as poping from an empty heap, or a heap with 1 item.
